### ISSUE TITLE
Windows Defender not opening on UTM Windows 11 App

### ISSUE THREAD

**Issue by mp31316** [utmapp/UTM#5126]
Windows Defender is not opening on the app. The following error appears if I try to open the WIndows Defender: "This app can't open: There's a problem with Windows Security. Reinstall the application from the original install location or contact your administrator". Please check the image below.

<img width="1180" alt="Screenshot 2023-03-16 at 4 00 24 PM" src="https://user-images.githubusercontent.com/28672289/225739327-f10b9ecc-6a08-4ec3-94f0-e5ad41acc527.png">

**Comment by derrickmanqele** [comment]
seeem to be having the same iisue myself

**Comment by akirataguchi115** [comment]
Same here

**Comment by IvanFengJK** [comment]
Same here. I am using MacBook Air using on M2 chip.

**Comment by sroui** [comment]
same here (macbook m1 2021)

**Comment by BitGrub** [comment]
Still an issue

**Comment by aptonline** [comment]
I don’t think this is a UTM specific issue as I’ve seen the same in VMware fusion so it’s likely a limitation of the arm version of Windows.
